Cargo.toml

```diff
@@ -15,4 +15,4 @@ borsh = { version = "0.9.3", default-features = false }
 default = ["mock-vm"]
 mock-vm = ["odra/mock-vm", "odra-modules/mock-vm"]
 casper = ["odra/casper", "odra-modules/casper"]
-
+e2e-tests = []
```

Comment:
`casper` is feature. So there is no need to define new.
Then run just: `cargo odra test -b casper`
And exclude test this way:
`#[cfg(all(test, not(feature = "casper")))]`